harmonica/equivalent_layer/harmonic.py

```diff
@@ -155,17 +187,54 @@ def fit(self, coordinates, data, weights=None):
         self.region_ = vd.get_region(coordinates[:2])
         coordinates = vdb.n_1d_arrays(coordinates, 3)
         if self.points is None:
-            self.points_ = (
-                coordinates[0],
-                coordinates[1],
-                coordinates[2] - self.relative_depth,
-            )
+            self.points_ = self.build_points(coordinates)
         else:
             self.points_ = vdb.n_1d_arrays(self.points, 3)
         jacobian = self.jacobian(coordinates, self.points_)
         self.coefs_ = vdb.least_squares(jacobian, data, weights, self.damping)
         return self
 
+    def build_points(self, coordinates):
```

Comment:
We probably want to make this private at first. If there is a need, we can always make it public later.